hail/python/hailtop/batch/batch_pool_executor.py

```diff
@@ -84,31 +84,31 @@ class BatchPoolExecutor:
 
     Parameters
     ----------
-    name: :obj:`str`, optional
+    name:
```

Comment:
I think you're missing the elimination of parameter types and return types in job.py.